src/CodeViewer/Viewer.cpp

```diff
@@ -329,16 +329,33 @@ void Viewer::DrawSampleCounters(QPaintEvent* event) {
   }
 }
 
+uint64_t Viewer::LargestOccuringLineNumber() const {
+  switch (line_number_types_) {
+    case LineNumberTypes::kNone:
+      return 0;
+    case LineNumberTypes::kOnlyMainContent:
+      return largest_occuring_line_number_main_content_.value_or(blockCount());
```

Comment:
Here blockCount() doesn't work?

Reply:
No. If we have annotating lines, then we have more blocks than main content lines.

Reply:
But here we are in OnlyMainContent case

Reply:
This is about line NUMBER types, not line types. We can still have both main content and annotating content, but we will only show line numbers for one of these cases. Or am I misunderstanding you?

Reply:
Yeah, I got it now

Reply:
Well, do you know what got you confused? Maybe we can make that more clear.